spec/factories/user.rb

```diff
@@ -4,6 +4,7 @@
     email 'person1@example.com'
     password 'password'
     password_confirmation { password }
+    locale 'ru'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.